lib/timers.js

```diff
@@ -94,8 +94,8 @@ const TIMEOUT_MAX = 2147483647; // 2^31-1
 //
 // - key = time in milliseconds
 // - value = linked list
-const refedLists = {};
-const unrefedLists = {};
+const refedLists = Object.create(null);
```

Comment:
For me it increases readability, but please check performance - I recall there being a regression when using `.create(null)` at one point.